comment-service/src/main/java/kostka/commentservice/CommentServiceApplication.java

```diff
@@ -4,13 +4,16 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
 
 @SpringBootApplication
+@EnableEurekaClient
+@EnableDiscoveryClient
```

Comment:
Tyto dve anotace jsou totozne - staci jen jedna pokud se nepletu. Pokud pouzijes `@EnableEurekaClient` tak jasne specifikujes, ze obsataravas service discovery pomoci Eureky. Pokud pouzije `@EnableDiscoveryClient`, tak ten hleda nejaky service discovery framework na classpath. Tj. v tvem pripade najde eureku a pouzije ji. Kazdopadne obe delaji to same si myslim.